app/upgrades/v20/upgrades.go

```diff
@@ -0,0 +1,85 @@
+package v20
+
+import (
+	"context"
+
+	providerkeeper "github.com/cosmos/interchain-security/v5/x/ccv/provider/keeper"
+
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+
+	"github.com/cosmos/gaia/v20/app/keepers"
+)
+
+// CreateUpgradeHandler returns an upgrade handler for Gaia v20.
+// It performs module migrations, as well as the following tasks:
+// - Initializes the MaxProviderConsensusValidators parameter in the provider module to 180.
+// - Sets the ValidatorSetCap parameter in the provider module to 180 for all presently registered consumer chains.
+// - Increases the MaxValidators parameter in the staking module to 200.
+func CreateUpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+	keepers *keepers.AppKeepers,
+) upgradetypes.UpgradeHandler {
+	return func(c context.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		ctx := sdk.UnwrapSDKContext(c)
+		ctx.Logger().Info("Starting module migrations...")
+
+		vm, err := mm.RunMigrations(ctx, configurator, vm)
+		if err != nil {
+			return vm, err
+		}
+
+		InitializeMaxProviderConsensusParam(ctx, keepers.ProviderKeeper)
```

Comment:
What about the `LastProviderConsensusValSet`? Shouldn't we set this during the upgrade?